src/main/java/com/mapzen/pelias/PeliasMapPositionListener.java

```diff
@@ -0,0 +1,6 @@
+package com.mapzen.pelias;
+
+public interface PeliasMapPositionListener {
```

Comment:
Upon closer inspection it looks like we dont need these changes. The only place we request the listener's lat/lng is in `suggest` so we can just update what the existing `PeliasLocationProvider` returns. Closing this PR